__tests__/buildOptionsList.test.js

```diff
@@ -0,0 +1,34 @@
+import { buildOptionsList } from "../src/buildOptionsList";
+import Icon from "react-native-vector-icons/AntDesign";
+
+describe("getting the list of options", () => {
```

Comment:
These tests don't make me confident that when they pass everything works fine. They are quite shallow. You can see some examples here https://github.com/th3rdwave/react-native-safe-area-context/tree/main/src/__tests__ and here https://github.com/th3rdwave/react-native-safe-area-context/tree/main/src/__tests__.

BTW, I don't see tests for the `Shkr` component.